recipes/decli/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "decli" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/decli-{{ version }}.tar.gz
+  sha256: f2cde55034a75c819c630c7655a844c612f2598c42c21299160465df6ad463ad
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - decli
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/decli/
+  summary: Minimal, easy-to-use, declarative cli tool
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - mxr-conda
```

Comment:
Forgot one thing: Someone from this account (or the account itself) would have to confirm that they want to maintain this package with you.

Reply:
Willing to maintain